### PR TITLE
dev-util/android-ndk: bump to 14b

### PR DESCRIPTION
**Pull request by pavelhomolka**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

**Comment by pavelhomolka** [comment]
https://bugs.gentoo.org/show_bug.cgi?id=616792

File: dev-util/android-ndk/android-ndk-14b.ebuild [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	ANDROID_PREFIX="${EPREFIX}/${ANDROID_NDK_DIR}"
+	ANDROID_PATH="${EPREFIX}/${ANDROID_NDK_DIR}"
+
+	for i in toolchains/*/prebuilt/linux-*/bin
+	do
+		ANDROID_PATH="${ANDROID_PATH}:${ANDROID_PREFIX}/${i}"
+	done
+
+	printf '%s' \
```
Comment (mgorny) [inline dev-util/android-ndk/android-ndk-14b.ebuild]: Is this a fancy way of saying `echo "foo"`?

Comment (pavelhomolka) [inline dev-util/android-ndk/android-ndk-14b.ebuild]: Yes. That's true. See e5d50763aff8eb78674e43c4d8116c40d7451030.

File: dev-util/android-ndk/android-ndk-14b.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	do
+		ANDROID_PATH="${ANDROID_PATH}:${ANDROID_PREFIX}/${i}"
+	done
+
+	echo "PATH=\"${ANDROID_PATH}\"" > "${T}/80${PN}" || die
+	doenvd "${T}/80${PN}"
+
+	echo "SEARCH_DIRS_MASK=\"${EPREFIX}/${ANDROID_NDK_DIR}\"" > "${T}/80${PN}" || die
+	insinto "/etc/revdep-rebuild"
+
```
Comment (mgorny) [inline dev-util/android-ndk/android-ndk-14b.ebuild]: I would also kill this empty line. It makes it quite confusing that you have echo and insinto in one block but the actual doins in another. But this is something that can be easily fixed while merging, so no need to push a fix.

**Comment by perfinion** [comment]
commit 38a2cd277aa59ac9ff690b0b0dc17d6555538064
Author: Pavel Homolka <pavelhomolka1986@gmail.com>
Date:   Mon Apr 24 02:58:26 2017

    dev-util/android-ndk: bump to 14b
    
    Gentoo-Bug: https://bugs.gentoo.org/616792
    
    Package-Manager: Portage-2.3.3, Repoman-2.3.1

Thanks for the patch! committed :D Sorry I didnt notice the PR earlier.

**Pull request closed without merging**